reana_workflow_controller/rest.py

```diff
@@ -521,103 +519,19 @@ def get_workflow_inputs(workflow_id):  # noqa
           description: Required. Workflow UUID.
           required: true
           type: string
-      responses:
-        200:
-          description: >-
-            Requests succeeded. The list of input files has been retrieved.
-          schema:
-            type: array
-            items:
-              type: object
-              properties:
-                name:
-                  type: string
-                last-modified:
-                  type: string
-                  format: date-time
-                size:
-                  type: integer
-        400:
-          description: >-
-            Request failed. The incoming data specification seems malformed.
-        404:
-          description: >-
-            Request failed. Workflow does not exist.
-          examples:
-            application/json:
-              {
-                "message": "Workflow 256b25f4-4cfb-4684-b7a8-73872ef455a1 does
-                            not exist."
-              }
-        500:
-          description: >-
-            Request failed. Internal controller error.
-          examples:
-            application/json:
-              {
-                "message": "Either organization or user doesn't exist."
-              }
-    """
-    try:
-        user_uuid = request.args['user']
-        user = User.query.filter(User.id_ == user_uuid).first()
-        if not user:
-            return jsonify(
-                {'message': 'User {} does not exist'.format(user)}), 404
-
-        workflow = Workflow.query.filter(Workflow.id_ == workflow_id).first()
-        if workflow:
-            outputs_directory = os.path.join(
-                current_app.config['SHARED_VOLUME_PATH'],
-                workflow.workspace_path,
-                current_app.config['INPUTS_RELATIVE_PATH'])
-
-            outputs_list = list_directory_files(outputs_directory)
-            return jsonify(outputs_list), 200
-        else:
-            return jsonify({'message': 'Workflow {} does not exist.'.
-                            format(str(workflow_id))}), 404
-
-    except KeyError:
-        return jsonify({"message": "Malformed request."}), 400
-    except Exception as e:
-        return jsonify({"message": str(e)}), 500
-
-
-@restapi_blueprint.route('/workflows/<workflow_id>/workspace/outputs',
-                         methods=['GET'])
-def get_workflow_outputs(workflow_id):  # noqa
-    r"""List all workflow output files.
-
-    ---
-    get:
-      summary: Returns the list of output files for a specific workflow.
-      description: >-
-        This resource is expecting a workflow UUID and a filename to return
-        its outputs.
-      operationId: get_workflow_outputs
-      produces:
-        - multipart/form-data
-      parameters:
-        - name: organization
-          in: query
-          description: Required. Organization which the worklow belongs to.
-          required: true
-          type: string
-        - name: user
+        - name: input_type
```

Comment:
The `get_workflow_files` endpoint is used to retrieve files (either type `code`, `input`, `output`) of a workflow / analysis.
The parameter `input_type` may not be the most descriptive term to specify that one wants to retrieve all files that have been created by a workflow.
Maybe `file_type` or just `type` would be more descriptive or not less confusing compared to `input_type`, but on the other hand `file_type` or `type` is kind of ambiguous.

Parameter `input_type` in `seed_workflow_workspace` endpoint doesn't need to be renamed since it's very descriptive in that context, but on the other hand, we could promote consistency and rename it to `file_type` or `type` as well.

Sorry I don't have better suggestions regarding to renaming the parameter...